src/PlcServer.cs

```diff
@@ -167,6 +169,22 @@ protected override void OnServerStarted(IServerInternal server)
 
         // request notifications when the user identity is changed, all valid users are accepted by default.
         server.SessionManager.ImpersonateUser += new ImpersonateEventHandler(SessionManager_ImpersonateUser);
+    }
+
+    /// <inheritdoc/>
+    protected override void ProcessRequest(IEndpointIncomingRequest request, object callData)
+    {
+        if (request is IAsyncResult asyncResult &&
+            asyncResult.AsyncState is object[] asyncStateArray &&
+            asyncStateArray[0] is TcpServerChannel channel)
+        {
+            using var scope = Logger.BeginScope("ChannelId:\"{ChannelId}\"", channel.Id);
```

Comment:
>\" [](http://example.com/codeflow?start=59&length=2)

Nit: add space after :

Reply:
no, the format goes directly in syslog and there is no space after :